src/cdk-experimental/listbox/listbox.ts

```diff
@@ -194,360 +158,382 @@ export class CdkOption<T = unknown> implements ListKeyManagerOption, Highlightab
     'role': 'listbox',
     'class': 'cdk-listbox',
     '[id]': 'id',
-    '(focus)': '_focusActiveOption()',
-    '(keydown)': '_keydown($event)',
-    '[attr.tabindex]': '_tabIndex',
+    '[attr.tabindex]': 'disabled ? null : 0',
```

Comment:
Not 100% sure on this, but I think if it's not using `activedescendant`, the `listbox` itself would be `tabindex="-1"` with the active option being `tabindex="0"`. Would need to confirm, though.

Reply:
I think that depends on how we want to handle the case where the listbox has not yet been interacted with. Currently its treated as having no active option (i.e. none of the items has the active styling). What happens when the user first tabs into an un-interacted listbox depends on if we're using aria-activedescendant or not:

For aria-activedescendant: when we tab into the un-interacted listbox focus goes to the listbox element and there is still no active option. Once we press down arrow the first option becomes active.

For roving tabindex: when we tab into the listbox the first option immediately becomes active and we land on it.

If we want to support this type of interaction model (which from what I can tell seems correct) we need the listbox itself to have tabindex=0 if there is no active option, even for the roving tabindex interaction. That gives the focus somewhere to land and we can then activate the option and pass focus along

Reply:
Yeah, I think that's fine for now and we can revisit in the future if necessary. Always having the listbox as `0` isn't _wrong_, but I'm not sure if it's optimal in all situations